service/src/Exception.cpp

```diff
@@ -103,6 +103,12 @@ namespace geopm
 
     }
 
+    Exception &Exception::operator=(const Exception &other)
+    {
+        m_err = other.m_err;
+        return *this;
+    }
```

Comment:
Should this call the base class' assignment operator as well?

Reply:
Thanks!